topics/imaging/tutorials/tutorial-CP/workflows/main_workflow.ga

```diff
@@ -1,1143 +1,1138 @@
+        "24": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/cp_cellprofiler/cp_cellprofiler/3.1.9",
+            "errors": null,
+            "id": 24,
+            "input_connections": {
+                "con_input_type|input_images": {
+                    "id": 1,
+                    "output_name": "output_tar"
+                },
+                "pipeline": {
+                    "id": 23,
+                    "output_name": "output_pipeline"
+                }
+            },
+            "inputs": [],
+            "label": "Run CellProfiler",
+            "name": "CellProfiler",
+            "outputs": [
+                {
+                    "name": "pipeline_output",
+                    "type": "input"
+                },
+                {
+                    "name": "logs",
+                    "type": "txt"
+                }
+            ],
+            "position": {
+                "left": 2053,
+                "top": 219
+            },
+            "post_job_actions": {
+                "EmailActionpipeline_output": {
+                    "action_arguments": {},
+                    "action_type": "EmailAction",
+                    "output_name": "pipeline_output"
+                },
+                "HideDatasetActionlogs": {
+                    "action_arguments": {},
+                    "action_type": "HideDatasetAction",
+                    "output_name": "logs"
+                }
+            },
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/cp_cellprofiler/cp_cellprofiler/3.1.9",
+            "tool_shed_repository": {
+                "changeset_revision": "fca368c99299",
+                "name": "cp_cellprofiler",
+                "owner": "bgruening",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"con_input_type\": {\"input_type\": \"yes\", \"__current_case__\": 1, \"input_images\": {\"__class__\": \"ConnectedValue\"}}, \"detailed_output\": \"true\", \"pipeline\": {\"__class__\": \"ConnectedValue\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "3.1.9",
+            "type": "tool",
+            "uuid": "5e893441-7a89-44ce-afe8-29959465f0ee",
+            "workflow_outputs": [
+                {
+                    "label": null,
+                    "output_name": "pipeline_output",
+                    "uuid": "30778509-8776-4c05-8d4e-6823532a9c41"
+                }
+            ]
         }
-      },
-      "label": "Run CellProfiler",
-      "inputs": [],
-      "position": {
-        "top": 364.86328125,
-        "left": 1059.94921875
-      },
-      "annotation": "",
-      "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/cp_cellprofiler/cp_cellprofiler/3.1.9",
-      "type": "tool"
-    }
-  },
-  "annotation": "Introduction to image analysis using Galaxy",
-  "a_galaxy_workflow": "true"
-}
\ No newline at end of file
+    },
```

Comment:
"tags": [
    "CellProfiler",
    "imaging"
  ],

I think....

Reply:
this is the workflow as I export it from Galaxy, did I do something wrong at an earlier stage?